catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -91,24 +110,24 @@ public String toString() {
         + securityService
         + ", catalogFramework="
         + catalogFramework
-        + ", filterBuilder="
-        + filterBuilder
         + ", workspaceTransformer="
         + workspaceTransformer
-        + ", filterService="
-        + filterService
+        + ", maxSubscriptions="
+        + maxSubscriptions.get()
```

Comment:
✏️ it would be mighty fine to clean this up into a string format or the like if you're feeling up to it

Reply:
done